[draft] Improve contrib and roadmaps doc (#1261)
f3d-app/f3d: README.md

# Support

F3D needs you help!

If you can, please consider sponsoring F3D, even a small amount, it would help us offseting the maintenance cost
as well as considering adding support for devices we do not own yet. Read more about it on our [sponsor page](https://github.com/sponsors/f3d-app).

If not, please use F3D, star it on github and share the word about it!

# Goals/Non-goals

We believe that clearly stating the goals and non-goals of F3D would help potential users and developers figure out if F3D is the right tool for them.

## Goals

- Support as many 3D file formats as possible

> - Support many types of visualization

Reviewer: ```suggestion
- Support many types of visualization (meshes, volumic, point clouds, ...)
```
for most people who are not from the sci.viz. community

> - Support all industries dealing with 3D datasets

Reviewer: I know what you mean but "industries" sounds a bit strong/specific; could it be reworded with something more like "use-case"/"scenario"/"workflow"/...?

- Let any user easily and quickly view any model with good defaults

> - Be as modular as possible to be built with a small number of dependencies
> - Be fully controllable from the command line and configuration file
> - Be usable non-interactively
> - Be as configurable as possible
> - Foster and inclusive and beginner friendly community

Reviewer: ```suggestion
- Be as configurable as possible
- Be fully controllable from the command line and configuration file
- Be usable non-interactively
- Foster and inclusive and beginner friendly community
- Be as modular as possible to be built with a small number of dependencies
```
reorder with the more "advanced" usage last?

# Support

F3D needs you help!

> If you can, please consider sponsoring F3D, even a small amount, it would help us offsetting the maintenance cost
> as well as considering adding support for devices we do not own yet. Read more about it on our [sponsor page](https://github.com/sponsors/f3d-app).

Reviewer: ```suggestion
If you can, please consider sponsoring F3D. Even a small donation would help us offsetting the recurring maintenance costs.
With enough sponsors we would even be able to add support for new devices (as we would need do acquire or rent them first). Read more about it on our [sponsor page](https://github.com/sponsors/f3d-app).
```
Clarify there's running costs, and you'd like to also be able to cover one-time costs (and what they would be, ie. access to hardware)

F3D needs you help!

> If you can, please consider sponsoring F3D. Even a small donation would help us offsetting the recurring maintenance costs.

Reviewer: ```suggestion
If you can, please consider sponsoring F3D. Even a small donation would help us offset the recurring maintenance costs.
```
I probably could have proof read my own suggestion better the first time around...
